src/organisms/Status/Illustration.constants.ts

```diff
@@ -0,0 +1,106 @@
+import { colors } from 'src/atoms';
+
+import { MotionProps } from 'framer-motion';
+
+interface Shape {
+  frames: string[];
+  fill: string;
+}
+
+export const SHAPES: Readonly<Shape[]> = [
+  {
+    /* vvvvvv   **/
```

Comment:
Any specific reason for this comment? (And another one further down in this file)

Reply:
No, removing them now

Reply:
Removing them now